kolibri/utils/env.py

```diff
@@ -41,9 +41,7 @@ def set_env():
     else.
     """
     from kolibri import dist as kolibri_dist  # noqa
-    sys.path = sys.path + [
-        os.path.realpath(os.path.dirname(kolibri_dist.__file__))
-    ]
+    sys.path = [os.path.realpath(os.path.dirname(kolibri_dist.__file__))] + sys.path
```

Comment:
oh right, I've been completely blind to this line having exactly the same issue :)